src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -146,13 +147,13 @@
         return this.getEventTypes(limiter, null);
     }
 
-    public Multi<EventType> getEventTypes(Query limiter, UUID applicationId) {
+    public Multi<EventType> getEventTypes(Query limiter, Set<UUID> applicationId) {
         String basicQuery = "SELECT et.id AS et_id, et.name AS et_name, et.description AS et_desc, a.id AS a_id, a.name AS a_name, a.description as a_description FROM public.event_type et " +
                 "JOIN public.application_event_type aet ON aet.event_type_id = et.id " +
                 "JOIN public.applications a ON a.id = aet.application_id";
 
-        if (applicationId != null) {
-            basicQuery += " WHERE a.id = $1";
+        if (applicationId != null && applicationId.size() > 0) {
+            basicQuery += " WHERE a.id = ANY ($1)";
```

Comment:
`= ANY` confused me for a moment as I would have used `IN` :)

Reply:
I tried `IN` initially, but didn't work (the query failed).